exampleCourse/questions/examplesRRNGSample/question.html

```diff
@@ -0,0 +1,31 @@
+<pl-question-panel>
+  <p>
+    With the seed of <b>{{params.seed}}</b>, generate a sample under replacement of size <b>{{params.sample_size}}</b> from a set of <b>{{params.set_size}}</b> integers.
```

Comment:
I’m not familiar with the terminology “sample _under_ replacement”. I would word this:
```suggestion
    With the seed of <b>{{params.seed}}</b>, generate a sample of size <b>{{params.sample_size}}</b> by sampling with replacement from a set of <b>{{params.set_size}}</b> integers.
```

Reply:
Done.